cmd/avalanche-network-runner/control/control.go

```diff
@@ -95,6 +98,12 @@ func newStartCommand() *cobra.Command {
 		"",
 		"[optional] JSON string of map that maps from VM to its genesis file path",
 	)
+	cmd.PersistentFlags().StringVar(
+		&nodeConfigFilePath,
+		"node-config-path",
```

Comment:
Should we just take in a flag `node-config` that expects a config JSON? (Discussed also on slack)

Reply:
ok done